docs/en/guides/nvidia-jetson.md

```diff
@@ -163,31 +183,75 @@ The YOLOv8n model in PyTorch format is converted to TensorRT to run inference wi
+
+### Comparison Chart
+
+Eventhough all model exports are working with NVIDIA Jetson, we have only included **PyTorch, TorchScript, TensorRT** for the comparison chart below because, they make use of the GPU on the Jetson and are guaranteed to produce the best results. All the other exports only utilize the CPU and the performance is not as good as the above three. You can find benchmarks for all exports in the section after this chart.
 
 <div style="text-align: center;">
     <img width="800" src="https://github.com/ultralytics/ultralytics/assets/20147381/202950fa-c24a-43ec-90c8-4d7b6a6c406e" alt="NVIDIA Jetson Ecosystem">
-</div>
-
-| Model   | Format      | Status | Size (MB) | mAP50-95(B) | Inference time (ms/im) |
-|---------|-------------|--------|-----------|-------------|------------------------|
-| YOLOv8n | PyTorch     | ✅      | 6.2       | 0.4473      | 14.3                   |
-| YOLOv8n | TorchScript | ✅      | 12.4      | 0.4520      | 13.3                   |
-| YOLOv8n | TensorRT    | ✅      | 13.6      | 0.4520      | 8.7                    |
-| YOLOv8s | PyTorch     | ✅      | 21.5      | 0.5868      | 18                     |
-| YOLOv8s | TorchScript | ✅      | 43.0      | 0.5971      | 23.9                   |
-| YOLOv8s | TensorRT    | ✅      | 44.0      | 0.5965      | 14.82                  |
-| YOLOv8m | PyTorch     | ✅      | 49.7      | 0.6101      | 36.4                   |
-| YOLOv8m | TorchScript | ✅      | 99.2      | 0.6125      | 53.34                  |
-| YOLOv8m | TensorRT    | ✅      | 100.3     | 0.6123      | 33.28                  |
-| YOLOv8l | PyTorch     | ✅      | 83.7      | 0.6588      | 61.3                   |
-| YOLOv8l | TorchScript | ✅      | 167.2     | 0.6587      | 85.21                  |
-| YOLOv8l | TensorRT    | ✅      | 168.3     | 0.6591      | 51.34                  |
-| YOLOv8x | PyTorch     | ✅      | 130.5     | 0.6650      | 93                     |
-| YOLOv8x | TorchScript | ✅      | 260.7     | 0.6651      | 135.3                  |
-| YOLOv8x | TensorRT    | ✅      | 261.8     | 0.6645      | 84.5                   |
-
-This table represents the benchmark results for five different models (YOLOv8n, YOLOv8s, YOLOv8m, YOLOv8l, YOLOv8x) across three different formats (PyTorch, TorchScript, TensorRT), giving us the status, size, mAP50-95(B) metric, and inference time for each combination.
+</div>   
+
+### Detailed Comparison Table
+
+The below table represents the benchmark results for five different models (YOLOv8n, YOLOv8s, YOLOv8m, YOLOv8l, YOLOv8x) across ten different formats (PyTorch, TorchScript, ONNX, OpenVINO, TensorRT, TF SavedModel, TF Graphdef, TF Lite, PaddlePaddle, NCNN), giving us the status, size, mAP50-95(B) metric, and inference time for each combination.
+
+
+??? Benchmarks
```

Comment:
With so many results, it may be optimal to use the tabbed admonition for the different model scales